llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -304,6 +310,10 @@ class MemoryDepChecker {
   /// restrictive.
   uint64_t MaxSafeVectorWidthInBits = -1U;
 
+  /// Maximum number of elements (power-of-2 and non-power-of-2), which do not
```

Comment:
Should this drop the `non-power-of-2` part or does it need other be added for `getStoreLoadForwardSafeVF`?